src/ShallowWrapper.js

```diff
@@ -142,6 +142,16 @@ class ShallowWrapper {
     this.complexSelector = new ComplexSelector(buildPredicate, findWhereUnwrapped, childrenOfNode);
   }
 
+  batchedUpdates(fn) {
+    const renderer = this.root.renderer;
+    if (renderer.unstable_batchedUpdates) {
+      // React 15.5+ exposes batching on shallow renderer itself
```

Comment:
should this use `REACT155` (or other version constants) so that it's only checked there?